### DISCUSSION TITLE
验收问题

### DISCUSSION THREAD

**Discussion by WT1W** [liusy58/NJU_AdvancedProgramming_SP22#64] [Q&A] [answered]
请问到时候是在自己的电脑上验收吗

**Comment by RicoloveFeng** [comment, marked as answer]
是的，并且验收的时候当场从cslabcms下载提交版本并运行，以免某些同学展示的时候用别人的代码，上传的时候却传了`三国演义.txt`😂

**Reply by ghost** [reply]
如何确保我和助教的编译环境相同呢？能否同时提交预编译的exe

**Reply by Price1999a** [reply]
> 如何确保我和助教的编译环境相同呢？能否同时提交预编译的exe

在你们自己电脑上当场编译验收。
